backfill/launchdarkly_react-native-client-sdk.json

```diff
@@ -0,0 +1,6 @@
+{
+  "version": 1,
+  "sdks": {
+    "react-native": {}
```

Comment:
`react-native` on purpose, the backfill works by mapping the old repo name (in the filename) to the current SDK ID.